core/views.py

```diff
@@ -113,30 +114,30 @@ def get_context_data(self, **kwargs):
                             output_field=IntegerField(),
                         )
                     ),
-                    sum_empty_net=Sum(
+                    sum_goals_against=Sum(
                         Case(
                             When(stat__team=F('roster__team'), stat__team__is_active=True,
-                                    then=Coalesce('stat__empty_net', 0)),
+                                    then=Coalesce('stat__goals_against', 0)-Coalesce('stat__empty_net', 0)),
                             default=0,
                             output_field=IntegerField(),
                         )
                     ),
-                    sum_goals_against=Sum(
+                    sum_games_played=Sum(
```

Comment:
I wonder if we need to eventually consider position, when figuring out games played. IE if someone usually plays another position, but every so often fills in for goalie. Definitely for a future pull request, just wondering.